lang/python/python-toml/Makefile

```diff
@@ -0,0 +1,43 @@
+#
+# Copyright (C) 2020 CZ.NIC, z. s. p. o. (https://www.nic.cz/)
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=python-toml
+PKG_VERSION:=0.10.1
+PKG_RELEASE:=1
+
+PYPI_NAME:=toml
+PKG_HASH:=926b612be1e5ce0634a2ca03470f95169cf16f939018233a670519cb4ac58b0f
+
+PKG_MAINTAINER:=Jan Pavlinec <jan.pavlinec@nic.cz>
+PKG_LICENSE:=MIT
+PKG_LICENSE_FILES:=LICENSE
+
+PKG_BUILD_PARALLEL:=0
+HOST_PYTHON3_PACKAGE_BUILD_DEPENDS:=setuptools-scm
```

Comment:
setuptools-scm [does not appear](https://github.com/uiri/toml/search?q=setuptools-scm&unscoped_q=setuptools-scm) to be needed.

Reply:
Removed